homeassistant/components/wink.py

```diff
@@ -5,13 +5,24 @@
 https://home-assistant.io/components/wink/
 """
 import logging
+import json
 
-from homeassistant.const import CONF_ACCESS_TOKEN, ATTR_BATTERY_LEVEL
 from homeassistant.helpers import validate_config, discovery
-from homeassistant.helpers.entity import ToggleEntity
+from homeassistant.const import CONF_ACCESS_TOKEN, ATTR_BATTERY_LEVEL
+from homeassistant.helpers.entity import Entity
 
 DOMAIN = "wink"
-REQUIREMENTS = ['python-wink==0.7.7']
+REQUIREMENTS = ['python-wink==0.7.8', 'pubnub==3.7.8']
```

Comment:
Python-wink doesn't handle the subscription to pubnub it only provides the details to home-assistant so it can do the subscription.